include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -588,7 +583,118 @@ namespace VectorTools
                 component_mask);
   }
 
+  template <int dim, int spacedim, typename VectorType>
+  void
+  interpolate(
+    const DoFHandler<dim, spacedim> &                          dof,
+    const Function<spacedim, typename VectorType::value_type> &function,
+    VectorType &                                               vec,
+    const ComponentMask &                                      component_mask)
+  {
+    if (dof.get_fe().has_support_points())
+      {
+        interpolate(StaticMappingQ1<dim, spacedim>::mapping,
+                    dof,
+                    function,
+                    vec,
+                    component_mask);
+      }
+    else
+      {
+        AssertDimension(vec.size(), dof.n_dofs());
+        const FiniteElement<dim, spacedim> &fe = dof.get_fe();
+
+        // Construct default fe_mask;
+        const ComponentMask fe_mask(
+          ComponentMask(fe.get_nonzero_components(0).size(), true));
```

Comment:
Any reason why this can't be
```suggestion
fe.get_nonzero_components(0).size(), true);
```
?

Reply:
Fixed, looks like an artifact from old code